.github/workflows/dotnetCourseTemplateTests.yml

```diff
@@ -17,10 +17,16 @@ jobs:
       with:
         dotnet-version: 6.0.x
 
-    - name: Restore dependencies
+    - name: Restore CustomExceptions dependencies
```

Comment:
I think this can be deleted

Reply:
The project will be built as a part of other solutions

Reply:
[https://github.com/podkolzzzin/AcademicHub/actions/runs/6905605591/job/18788765781#step:4:266](url)